diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/chapter6/classes/CompactInitialization.kt

```diff
@@ -145,7 +147,24 @@ class CompactInitialization(configRules: List<RulesConfig>) : DiktatRule(
         ?: run {
             // add apply block
             property.node.run {
-                val newInitializerNode = kotlinParser.createNode("${property.initializer!!.text}.apply {}")
+                val isNotRequiresParentheses = property.initializer.let {
```

Comment:
May be we will move this logic with adding apply block to some new method...
If it looks ok

Reply:
Extracted